### PR TITLE
[Docker][G5] add test artifacts

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add multi-gpu tests on the large model inference

### CODE REVIEW

File: tests/integration/deepspeed/deepspeed-model.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from djl_python import Input, Output
+import deepspeed
+import torch
+from transformers import pipeline, AutoModelForCausalLM, AutoTokenizer
+
+torch.manual_seed(1234)
+
+def load_model(properties):
+    tensor_parallel = properties["tensor_parallel_degree"]
+    model = AutoModelForCausalLM.from_pretrained(properties["model_path"])
```
Comment (frankfliu) [inline tests/integration/deepspeed/deepspeed-model.py:10]:
```suggestion
    model = AutoModelForCausalLM.from_pretrained(properties["model_dir"])
```

File: tests/integration/deepspeed/prepare.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import argparse
+import os
+import shutil
+
+parser = argparse.ArgumentParser(description='Build the LLM configs')
+parser.add_argument('model',
+                    help='s3 bucket url')
+
+model_list = {
+    "opt-30b" : { "s3url" :"s3://djl-llm/opt-30b/", "tensor_parallel_degree" : 4, "model_path" : "/opt/ml/model/opt-30b" }
```
Comment (frankfliu) [inline tests/integration/deepspeed/prepare.py:10]: You don't need define model_path, "model_dir" is built-in property

Comment (frankfliu) [inline tests/integration/deepspeed/prepare.py:10]:
```suggestion
    "opt-30b" : { "engine": "DeepSpeed", "option.entryPoint": "djl_python.deepspeed",  "option.s3url" :"s3://djl-llm/opt-30b/", "tensor_parallel_degree" : 4 }
```

File: tests/integration/deepspeed/deepspeed-model.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+from djl_python import Input, Output
```
Comment (frankfliu) [inline tests/integration/deepspeed/deepspeed-model.py:1]: Why not use built-in deepspeed.py?

Comment (lanking520) [reply tests/integration/deepspeed/deepspeed-model.py:1]: need some customization on tests and also build batches.

File: tests/integration/deepspeed/prepare.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import argparse
+import os
+import shutil
+
+parser = argparse.ArgumentParser(description='Build the LLM configs')
+parser.add_argument('model',
+                    help='s3 bucket url')
+
+model_list = {
+    "opt-30b" : { "option.s3url" :"s3://djl-llm/opt-30b/", "tensor_parallel_degree" : 4, "model_id" : "opt-30b" }
```
[the remark this answers is not in the document]
Comment (lanking520) [reply tests/integration/deepspeed/prepare.py:10]: Yes, but without telling which model is, how will the model.py find that opt-30b folder?

**Review by frankfliu — APPROVED** [review]

**Pull request merged**